### ISSUE TITLE
USWDS-Site - Add what's new content to the home page

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds-site#2891]
## Summary
Add featured content from the what's new page to the home page.

**Comment by mejiaj** [comment]
We'll need to update the weight/complexity before starting.

**Comment by dct2023** [comment]
@amyleadem Can you share the status of this one? I want to make sure that it's slated for an upcoming sprint with the details needed to work on it.

**Comment by dct2023** [comment]
Note: This was slated as a nice to have for the demo in February if we have bandwidth.

**Comment by dct2023** [comment]
Added to upcoming sprint but once we get more information from Amy we can adjust if needed.

**Comment by amyleadem** [comment]
Converting this issue to be a follow-up to the wireframe work in #3106.
